src/pages/Wishlist/Wishlist.jsx

```diff
@@ -1,76 +1,79 @@
-              </div>
+        ) : (
+          <div>
+            <div className="wishlist-header">
+              <h3>My Wishlist {wishlist.length}</h3>
             </div>
-            <div className="action-items">
-              <div className="move-to-cart">
-                <button className="btn primary-btn-outline">
-                  Move To Cart
-                </button>
-              </div>
-              <div className="delete-item">
-                <span className="delete-icon">
-                  <i className="fa-solid fa-trash"></i>
-                </span>
-              </div>
+            <div className="border-bottom"></div>
+            <div className="wishlist-items-container">
+              {wishlist.map((product) => {
+                return (
+                  <div>
+                    <div className="wishlist-item" key={product._id}>
+                      <div className="item-image">
+                        <img
+                          className="responsive-img"
+                          src={product.img}
+                          alt="product"
+                        />
+                      </div>
+                      <div className="item-details">
+                        <div className="item-title">{product.title}</div>
+                        <div className="item-subtitle">By {product.author}</div>
+                        <div className="card-price">
+                          <span className="price-now">
+                            {" "}
+                            ₹{product.discountedPrice}
+                          </span>
+                          <span className="price-before">₹{product.price}</span>
+                          <span className="discount">
+                            {" "}
+                            {getDiscountPercent(
+                              product.price,
+                              product.discountedPrice
+                            )}
+                            %
+                          </span>
+                        </div>
+                      </div>
+                      <div className="action-items">
+                        <div className="move-to-cart">
+                          <button className="btn primary-btn-outline">
+                            Move To Cart
+                          </button>
+                        </div>
+                        <div
+                          className="delete-item"
+                          onClick={() =>
+                            removeFromWishlist(product, productDispatch)
+                          }
+                        >
+                          <span className="delete-icon">
+                            <i className="fa-solid fa-trash"></i>
+                          </span>
+                        </div>
+                      </div>
+                    </div>
+                    <div class="border-bottom"></div>
+                  </div>
```

Comment:
Typo here

Reply:
I have corrected it now, thanks Jowel for catching it!